server/index.js

```diff
@@ -0,0 +1,23 @@
+const express = require('express');
+const app = express();
+const path = require('path');
+//const routes = require('./routes/routes');
```

Comment:
perhaps you can use routes/index.js if multiple `/api/${model}` routes get made or bring in each individually